src/avk.cpp

```diff
@@ -4431,12 +4484,16 @@ namespace avk
 
 		graphics_pipeline_t result;
 
-		// 0. Own the renderpass
+		// 0. Own the renderpass - if one is required
+		const bool dynamic_rendering_enabled = aConfig.mDynamicRendering == avk::cfg::dynamic_rendering::enabled;

```

Comment:
```suggestion
		const bool dynamicRenderingEnabled = aConfig.mDynamicRendering == avk::cfg::dynamic_rendering::enabled;

```

Please use lowerCamelCase for local variables!